src/modules/OpenChannel/components/OpenChannelMessage/index.tsx

```diff
@@ -114,116 +99,130 @@ export default function MessagOpenChannelMessageeHoc(props: OpenChannelMessagePr
           updateMessage(messageId, message);
           setShowEdit(false);
         }}
-        onCancelEdit={() => { setShowEdit(false); }}
+        onCancelEdit={() => {
+          setShowEdit(false);
+        }}
       />
     );
   }
 
   return (
     <div className="sendbird-msg-hoc sendbird-msg--scroll-ref">
-      {/* date-separator */}
-      {
-        (hasSeparator && message?.createdAt) && (
+      <>
```

Comment:
why is this needed?

Reply:
<img width="788" alt="image" src="https://github.com/sendbird/sendbird-uikit-react/assets/26326015/66f292c5-ee79-4841-8a79-25d5fd3584cd">
It's because of this error.
Perhaps the code is quite complex, and the compiler doesn't seem to recognize it correctly.
That's why I simply wrapped it with a Fragment